src/averageAges.js

```diff
@@ -56,6 +77,25 @@ function calculateWomenAverageAge(people, withChildren) {
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
   // write code here
+  const peopleWithAgeDiff = people.map((person, i, array) => {
+    const momIndex = array.findIndex(a => a.name === person.mother);
+
+    if (momIndex !== -1) {
+      person.ageDiff = person.born - array[momIndex].born;
+    }
```

Comment:
Instead of searching of index - you can find object using `find`, then you can use it instead of using index to find needed object

Reply:
Also - it's a bad practice to mutate incoming objects